hack/lib/util.sh

```diff
@@ -119,17 +119,13 @@ kube::util::wait-for-jobs() {
   return ${fail}
 }
 
-# takes a binary to run $1 and then copies the results to $2
+# Takes a binary to run $1 and then copies the results to $2.
+# If the generated and original files are the same after filtering lines
+# that match $3, copy is skipped.
```

Comment:
Just one thing is I think you could improve documentation here to indicate what $3 is exactly. Some examples might help. Though now it's fairly clear from the code, so I think someone reading the code will understand what the intent is here.